googletest/cmake/internal_utils.cmake

```diff
@@ -188,6 +188,10 @@ function(cxx_library_with_type name type cxx_flags)
     endif()
     target_link_libraries(${name} PUBLIC ${threads_spec})
   endif()
+
+  if (NOT "${CMAKE_VERSION}" VERSION_LESS "3.8")
```

Comment:
I'm wondering if we can help out users using CMake < 3.8 with an else branch. Maybe something like `cxx_auto_type`, `cxx_nullptr`, and `cxx_static_assert` to effective say C++11 for those users?

Reply:
Or check that user defined `CMAKE_CXX_STANDARD` greater than 11 ? If not issue a warning saying build might break

Reply:
I'm happy accepting this as is. We can iterate if people complain about older cmake versions.